garden_ai/scripts/save_session_and_metadata.py

```diff
@@ -31,6 +31,37 @@ def assert_compatible_dill_version():
         raise EnvironmentError(message)
 
 
+def get_requirements_file():
+    """Return pathlib.Path to a requirements file in the container if one exists.
+
+    Requirements are written into the container in the same directory as the notebook.
+    See garden_ai/app/notebook.py `publish`.
+    """
+    from pathlib import Path
+
+    pip = Path(__file__).parent / "requirements.txt"
+    if pip.exists():
+        return pip
+
+    conda = Path(__file__).parent / "requirements.yml"
```

Comment:
easy to be misled because I think we still have some conda-related cruft laying around the repo, but we actually don't currently support conda in the base images so we only need to worry about pip